go/vt/vtgate/planbuilder/operators/projection.go

```diff
@@ -201,16 +201,14 @@ func createSimpleProjection(ctx *plancontext.PlanningContext, selExprs []sqlpars
 // been settled. Once they have settled, we know where to push the projection, but if we push too early
 // the projection can end up in the wrong branch of joins
 func (p *Projection) canPush(ctx *plancontext.PlanningContext) bool {
-	if reachedPhase(ctx, subquerySettling) {
-		return true
-	}
+	subQSettled := reachedPhase(ctx, subquerySettling)
```

Comment:
Mind renaming this to `allSubqueriesSettled`? Single character abbreviations make sense in variable names when familiar with the code, but when reading this with "fresh eyes" it's easier to understand if variable names are not abbreviated. 🙇‍♂️